app/styles/main.css

```diff
@@ -0,0 +1,13 @@
+@import '@fontsource/ubuntu-mono';
```

Comment:
When I run the build command, it generates several font files, although it looks like `ubuntu-mono-latin-400-normal` is the only font that actually gets loaded on the page.

While this import might be a little simpler and more forward-compatible, I feel like I'd rather this be:

```
@import '@fontsource/ubuntu-mono/latin-400.css';
```

Just because it would simplify the build output by not generating font files for fonts that will never be loaded.

Reply:
@jacksonrayhamilton Fixed in e8fc5b6b690b2e2e20c73bce8375e900b16b9684

Reply:
Thanks!